app/models/concerns/user_two_factor_auth.rb

```diff
@@ -0,0 +1,61 @@
+require 'active_support/concern'
+
+module UserTwoFactorAuth
+  extend ActiveSupport::Concern
+
+  included do
+  	attr_accessor :two_factor
+  	devise :two_factor_authenticatable, :two_factor_backupable,
+         :otp_secret_encryption_key => ENV['TWO_FACTOR_KEY']
```

Comment:
As we discussed on Slack, please replace `ENV['TWO_FACTOR_KEY']` by `CONFIG['two_factor_key']` (and thus add `two_factor_key` to `config.yml`)

Reply:
Fixed